docs/glossary/terms.md

```diff
@@ -0,0 +1,226 @@
+
+Spore is a special form of NFT where the content is entirely and permanently stored on-chain, intrinsically tied to the token value.
+
+**see also:** 
+* [Is Spore an NFT?](https://spore-docs-9bn3yjbww-sporeprotocol.vercel.app/basics/tech-faq#is-spore-an-nft)
+
+## Omnilock
+
+An interoperable lock script natively supporting transaction signing method verification across blockchains, offering extensibility for future verification algorithms. 
+
+Omnilock introduces a regulation compliance module, enabling administrators to revoke tokens held by users under defined circumstances, bridging the gap between blockchain and traditional financial regulation. 
+
+For more information, visit [RFC0042-Omnilock](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0042-omnilock/0042-omnilock.md).
+
+## output
+
+Spore cell(s) created when new transactions are confirmed. Can serve as input cells for future transactions.
+
+**see also:** 
+* [input](#input)
+
+## on-chain ownership
+
+Publicly verifiable and indisputable record of ownership for digital assets inscribed on the blockchain, eliminating the need for intermediaries and ensuring transparency.
+
+## private cluster
+
+Cluster permitting only the owner to associate a Spore or multiple Spores when minting.
+
+**see also:** 
+* [public cluster](#public_cluster)
+* [cluster](#cluster)
+
+## private key
+
+Cryptographic information exclusively known to its owner or owners. Employed in conjunction with an algorithm for both encryption and decryption, ensuring the confidentiality and security of sensitive information.
+
+**see also:** 
+* [public key](#public_key)
+
+## public cluster
+
+Cluster enabling anyone, besides the owner, to associate a Spore or multiple Spores when minting.
+
+**see also:**
+* [cluster](#clsuter)
+* [private cluster](#private_cluster)
+
+## public key
+
+Cryptographic information openly shareable without compromising security. Utilized for message encryption to ensure that only the corresponding private key holder can decrypt. Also employed for message authenticity verification, confirming authorization by the user possessing the corresponding private key.
+
+**see also:** 
+* [private key](#public_key)
+
+## redeemability
+
+Ability of Spore users to destroy their Spores and instantly recover the remaining value at no cost. 
+
+**see also:**
+* [Why is Spore‘s value redeemable?](https://docs.spore.pro/basics/tech-faq#why-is-spores-value-redeemable)
+
+## secp256k1-blake160
+
+A lock script code using the [secp256k1](https://en.bitcoin.it/wiki/Secp256k1) signature algorithm, performing [Blake2b](https://www.blake2.net/) hash operations on transaction and witness components. Essential for validating transactions in specific blockchain systems.
+
+## transfer
+
+Process that refreshes a Spore by consuming its cell and creating a new one with the same Spore ID and unchanged fields. This operation costs no additional free, where the costs is deducted from the cell's capacity.
```

Comment:
```suggestion
The process of changing the ownership of a Spore or a Cluster. It's important to note that owning a cluster does not automatically grant ownership of the spores associated with that cluster. Each Spore and Cluster has its own distinct ownership rights that can be transferred separately.
```
the implementation bits should be sufficient under "see also", especially the information is the same as basics/spore-operation#transfer